src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -33,3 +34,67 @@ def test_property_real64(multi_instrument_session):
 def test_property_string(multi_instrument_session):
     multi_instrument_session.start_label = 'foo'
     assert multi_instrument_session.start_label == 'foo'
+
+
+def test_tdr_all_channels(multi_instrument_session):
+    applied_offsets = multi_instrument_session.tdr(apply_offsets=False)
+    assert len(applied_offsets) == multi_instrument_session.channel_count
+
+    multi_instrument_session.apply_tdr_offsets(applied_offsets)
+
+    channels = [multi_instrument_session.get_channel_name(i) for i in range(1, multi_instrument_session.channel_count + 1)]
+    fetched_offsets = [multi_instrument_session.channels[i].tdr_offset for i in channels]
+    assert fetched_offsets == applied_offsets
+
+
+def test_tdr_some_channels(multi_instrument_session):
+    channels = [multi_instrument_session.get_channel_name(i) for i in [64, 1, 50, 25]]
+    applied_offsets = multi_instrument_session.channels[channels].tdr(apply_offsets=False)
+    assert len(applied_offsets) == len(channels)
+
+    multi_instrument_session.channels[channels].apply_tdr_offsets(applied_offsets)
+
+    fetched_offsets = [multi_instrument_session.channels[i].tdr_offset for i in channels]
+    assert fetched_offsets == applied_offsets
+
+
+def test_source_waveform_parallel_broadcast(multi_instrument_session):
+    configure_session(multi_instrument_session)
+
+    multi_instrument_session.create_source_waveform_parallel(
+        pin_list='LowPins',
+        waveform_name='new_waveform',
+        data_mapping=2600)
+
+    multi_instrument_session.write_source_waveform_broadcast_u32(
+        waveform_name='new_waveform',
+        waveform_data=[i for i in range(4)])
+
+    multi_instrument_session.burst_pattern(
+        site_list='',
+        start_label='new_pattern',
+        select_digital_function=False,
+        wait_until_done=True,
+        timeout=5)
+
+    pass_fail = multi_instrument_session.get_site_pass_fail(site_list='')
+    assert pass_fail == [True, True]
+
+
+def configure_session(session):
+    test_files_dir = os.path.join(os.getcwd(), 'src\\nidigital\\system_tests\\test_files')
```

Comment:
You should use separate strings for each part of the path. I.e.
``` python
test_files_dir = os.path.join(os.getcwd(), 'src', 'nidigital', 'system_tests', 'test_files')
```

Reply:
Done

Reply:
Not all OSes use backslashes.